azure-mgmt-network/HISTORY.rst

```diff
@@ -3,11 +3,36 @@
 Release History
 ===============
 
-1.8.0a1 (2017-XX-XX)
-++++++++++++++++++++
+2.0.0rc1 (2018-03-07)
++++++++++++++++++++++
+
+**Breaking changes**
+
+This version introduces a new generation of code generator that *might* introduce breaking changes. 
+
+- Model signatures are now using only keywords-arguments syntax. Every positional arguments are required to be rewritten as keywords-arguments.
+  To keep auto-completion in most cases, models are now generated for Python 2 and Python 3. Python 3 uses the "*" syntax for keyword-only arguments.
+- Enum type are now using the "str" mixin (`class AzureEnum(str, Enum)`) to improve experiences when unkown enum are met. This is not a breaking change,
```

Comment:
Enum types now use the "str" mixin (`class AzureEnum(str, Enum)`) to improve the behavior when unrecognized enum values are encountered.